<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadProgress</class>
 <widget class="QWidget" name="DownloadProgress">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QProgressBar" name="decryptionProgressBar">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>331</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>73</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Decryption Progress:</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="downloadProgressBar">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>331</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>23</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Download Progress:</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="extractionProgressBar">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>331</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>122</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Extraction Progress:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
